gui/default/index.html

```diff
@@ -32,11 +32,11 @@
 
   <nav class="navbar navbar-top navbar-default" role="navigation">
     <div class="container">
-      <span class="navbar-brand">
+      <span class="navbar-brand" aria-hidden="true">
         <img class="logo hidden-xs" src="assets/img/logo-horizontal.svg" height="32" width="117"/>
         <img class="logo hidden visible-xs" src="assets/img/favicon-default.png" height="32"/>
       </span>
-      <p class="navbar-text hidden-xs" ng-class="{'hidden-sm':upgradeInfo && upgradeInfo.newer}">{{thisDeviceName()}}</p>
+      <h1 class="navbar-text hidden-xs" ng-class="{'hidden-sm':upgradeInfo && upgradeInfo.newer}">{{thisDeviceName()}}</h1>
```

Comment:
This change doesn't seem good though, it increases the size of the device name in the header drastically. Perhaps there is an aria tag that could be used instead to indicate it's headerness, or the styles need to be tweaked to not do this for h1-in-navbar.

<img width="999" alt="screen shot 2016-07-30 at 10 08 02" src="https://cloud.githubusercontent.com/assets/125426/17269031/8fa5d484-563d-11e6-8c4c-3aea7e582e1e.png">